src/NServiceBus.Core/Receiving/ReceiveComponent.Configuration.cs

```diff
@@ -22,19 +22,20 @@ public static Configuration PrepareConfiguration(HostingComponent.Configuration
             var purgeOnStartup = settings.PurgeOnStartup;
 
             var transportDefinition = transportSeam.TransportDefinition;
-            var localAddress = transportDefinition.ToTransportAddress(new QueueAddress(queueNameBase, null, null, null));
+            var localAddress = transportDefinition.ToTransportAddress(new QueueAddress(queueNameBase));
```

Comment:
In theory we could get rid of this and always perform this resolution in the extension methods?